.github/workflows/node-ci.yml

```diff
@@ -48,10 +50,13 @@ jobs:
         run: yarn generate
 
       - name: Lint code
-        run: yarn lint --ignore-pattern e2e/
+        run: yarn lint
 
       - name: Run Jest tests
         run: yarn test --coverage --silent
 
-      - name: Build Storybook
-        run: yarn storybook --ci --smoke-test
+      - name: Build Storybook and start Happo run
```

Comment:
Happo command will build story book before running tests. The old `yarn storybook --ci --smoke-test` basically built storybook and exited. So to avoid building storybook twice I replaced it.